src/Security/Group.php

```diff
@@ -115,6 +115,11 @@ public function getAllChildren()
      */
     public function getCMSFields()
     {
+        $list = Group::get()->exclude('ID', $this->ID);
+        $groups = new ArrayList();
+        foreach ($list as $grp) {
+            $groups->push(['ID' => $grp->ID, 'Title' => html_entity_decode($grp->Breadcrumbs)]);
+        }
```

Comment:
You could probably put this into a private method, e.g. `$this->getDecodedBreadcrumbs()` which contains this logic